tracing/plugin/otlp_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+// TestNewExporter runs tests with different combinations of configuration for NewExporter function
+func TestNewExporter(t *testing.T) {
```

Comment:
wonder if it's a good idea to write a test for the newTracer function?

Reply:
That's what I was debating myself.   
I tried to write a test for newTracer then realized that function depends on [TracingProcessorPlugin](https://github.com/containerd/containerd/blob/main/tracing/plugin/otlp.go#L157) , registered first.  
I think that should be treated as integration test I am going to add.